app/src/presenters/analysisResultsPresenter.js

```diff
@@ -90,6 +91,11 @@ const decorateWithArea = (results, subscription) => {
     return results;
 };
 
+const decorateWithCustomImage = async (results, subscription) => {
+    results.alert_custom_image = await ImageService.getStaticMapImageUrl(subscription);
```

Comment:
the email templates expect `image_url` for the area. Is this mapped somewhere else?

Reply:
No no, this was added now, gonna change to the name the template expects!